docs/authentication/social-connections/tiktok.mdx

```diff
@@ -1,47 +1,75 @@
 
-Adding social connection with TikTok to your app with Clerk is done in a few steps - you only need to set the **Client ID**, **Client Secret** and **Authorized redirect URI** in your instance settings.
+> [!IMPORTANT]
+> Due to TikTok's requirement to verify URL ownership for all redirect URLs, TikTok cannot be used as a social connection in development environments. This is because individual users cannot verify ownership of the Clerk development URL (`accounts.dev`). However, TikTok can be used properly in production environments where you can verify your own URLs. For more information, see [TikTok's URL ownership guidelines](https://developers.tiktok.com/doc/getting-started-create-an-app#verify_url_ownership).
 
-To make the development flow as smooth as possible, Clerk uses  preconfigured shared OAuth credentials and redirect URIs for development instances - no other configuration is needed.
+To make the setup process easier, it's recommended to keep two browser tabs open: one for your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [TikTok Developer Portal](https://developers.tiktok.com/).
 
-For production instances, you will need to create your own developer account with **TikTok** and generate your own Client ID and Client secret.
+<Steps>
+  ### Enable TikTok as a social connection in Clerk
 
-> [!NOTE]
-> The purpose of this guide is to help you set up a TikTok developer account  and a TikTok OAuth app - if you're looking for step-by-step instructions using Clerk to add social connection (OAuth) to your application, follow the [Social connection (OAuth) guide](/docs/authentication/social-connections/oauth).
+  1. In the Clerk Dashboard, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
+  1. Select **Add connection** and select **For all users**.
+  1. In the **Choose provider** dropdown, select **TikTok**.
+  1. Ensure that both **Enable for sign-up and sign-in** and **Use custom credentials** are toggled on.
+  1. Save the **Authorized redirect URI** somewhere secure. Keep the modal and page open.
 
-## Before you start
+  ### Create your app in TikTok
 
-- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application guide](/docs/quickstarts/setup-clerk).
-- You need to have a TikTok Developer account. To create one, visit the [TikTok for developers](https://developers.tiktok.com/) page and click on Login ➜ Sign Up
+  > [!TIP]
+  > If you're creating an app as an organization rather than as an individual developer, you must first [create an organization](https://developers.tiktok.com/organizations). For guidance on which option to choose, refer to the [Working with organizations guide](https://developers.tiktok.com/doc/working-with-organizations/).
```

Comment:
Let's specify that it's TikTok's guide, and remove the exact guide title incase it ever changes.

```suggestion
  > If you're creating an app as an organization rather than as an individual developer, you must first [create an organization](https://developers.tiktok.com/organizations). For guidance on which option to choose, refer to [TikTok's guide on working with organizations](https://developers.tiktok.com/doc/working-with-organizations/).
```